### PR TITLE
Spelling, Formating, and other minor changes

### PR DESCRIPTION
**Pull request by mcdonnelldean**
@mcollina Some changes on my _night off_ from coding :D Just don't tell the other half :D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,58 +262,56 @@ Connects this HyperEmitter with another one which we call a peer. Peers can exis
...
+  * `from:` The point in the stream to return events since. supports 'beginning'
 
 <a name="close"></a>
 ### .close(callback)
-Close the given __hyperemitter__. After, all `emit` will return an error.
+Close a given HyperEmitter. After, all `emit` will return an error.
 
-##### _callback_
-An optional function to be called when close has completed.
+ * #### _callback_
```
Comment (mcollina) [inline README.md]: I'm not sure why, but this does not render well in Github.

**Comment by mcollina** [comment]
BTW, I gave you the commit bit, you can send PRs right from this repo ;).

File: README.md (hunk 1 above)
Comment (mcdonnelldean) [inline README.md]: @mcollina Specifically the size? They looked a lot smaller in my markdown preview, let me make them a smaller.

**Comment by mcdonnelldean** [comment]
@mcollina Yeah I seen that. I do like to maintain my own fork though, it just means any silly pushes to origin on my part can be nuked by reforking :D

**Comment by mcdonnelldean** [comment]
@mcollina That was really odd! A hard space fixed the issue. I made the args smaller as well, they didn't look that big in my previewer!

**Pull request merged**